### DISCUSSION TITLE
Insufficient memory (case 4) `c:\\0002\0002\0026.jpg' @ error/jpeg.c/JPEGErrorHandler/347

### DISCUSSION THREAD

**Discussion by jueqing1015** [dlemstra/Magick.NET#1027] [Help]
Insufficient memory (case 4) `c:\\0002\0002\0026.jpg' @ error/jpeg.c/JPEGErrorHandler/347
Process the image to prompt this. What's the problem, please

**Comment by dlemstra** [comment]
Can you please share the file that causes this issue?

**Reply by jueqing1015** [reply]
Sorry, the image cannot be sent out because it is internal data

The details are as follows: there are about thousands of images in my side in cyclic processing, and they will be released after each run, but this error will be reported after more than 100 images are basically run

  报错

**Reply by dlemstra** [reply]
My guess is that you are not disposing the images and you are running out of memory.

**Reply by jueqing1015** [reply]
@dlemstra 
![捕获](https://user-images.githubusercontent.com/65677359/133555671-3a8da00e-4c05-4f2f-adca-2fb632a5a478.JPG)
![捕获](https://user-images.githubusercontent.com/65677359/133555676-d9455554-b86f-4415-8a44-d51e2036e276.PNG)

**Reply by dlemstra** [reply]
Did you check how much memory your application is using? I really think you are keeping too many images in memory.
